packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -91,6 +94,8 @@ const parsedSubscriberDestination: DestinationWithoutToken = {
   url: 'http://subscriber.com'
 };
 
+const mockedDestinationUrlSubaccountDestinations =
+  'https://destination.example.com/destination-configuration/v1/subaccountDestinations';
```

Comment:
[req] Can you import the `destinationServiceUri` from test util. I wasn't sure looking at it where the url came from

Reply:
I imported `destinationServiceUri`, but the path `/destination-configuration/v1/subaccountDestinations` still has to be appended as I didn't find the path in constants.